command/results_xunit.go

```diff
@@ -162,3 +152,39 @@ func (c *xunitResults) parseAndUploadResults(ctx context.Context, conf *model.Ta
 
 	return sendJSONResults(ctx, conf, logger, comm, &task.TestResults{tests})
 }
+
+func generateLogsForOneFile(testSuites []testSuite, t *task.Task, tests []task.TestResult, logs []*model.TestLog, logIdxToTestIdx []int) ([]task.TestResult, []*model.TestLog, []int) {
```

Comment:
there's so much about this function signature that's frustrating, but I think it's better to have it separated.

Reply:
I think you have to send test logs all at once? regardless, I think it's fine as is.